seed/seed.py

```diff
@@ -0,0 +1,25 @@
+import psycopg2
+
+conn = psycopg2.connect(
+    database="soengram",
+    user="postgres",
+    password="admin",
+    host="localhost",
+    port="5432"
+)
+
+cur = conn.cursor()
+
+cur.execute('''
+            CREATE TABLE "user" (
+                user_id SERIAL PRIMARY KEY,
+                email VARCHAR(255) UNIQUE NOT NULL,
+                username VARCHAR(255) UNIQUE NOT NULL,
```

Comment:
This seems a bit excessive. Surely we don't want usernames to be up to 255 in length.

Reply:
dont think it matters we will do the validation on the FE we wont leave it up to the DB

Reply:
Validation should be done on the backend for security reasons. For example, I could turn off/block JS before form submission and then none of the data would be validated at that point.

Reply:
updated